packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -44,47 +45,55 @@ keywords: [combobox, aria]
 
 ## Features
 
-A ComboBox can be built using the [&lt;datalist&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) HTML element, but this is very limited in functionality and difficult to style.
-`useComboBox` helps achieve accessible ComboBox components that can be styled as needed.
+A combo box can be built using the [&lt;datalist&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) HTML element, but this is very limited in functionality and difficult to style.
```

Comment:
What do you think about making the `useCombobox/useComboBoxState` description and other type descriptions use `combo box` instead of `ComboBox`? Just thinking about consistency here.

I can go ahead and do this in my PR if you feel its reasonable